docker/kb-importer/Dockerfile

```diff
@@ -15,10 +15,13 @@ WORKDIR /kb-importer
 RUN wget https://github.com/SAP/project-kb/releases/download/v0.6.18/kaybee-0.6.18_linux-amd64 -O kaybee
 RUN chmod +x kaybee
 
-COPY kb-importer-$VULAS_RELEASE-jar-with-dependencies.jar kb-importer.jar
+COPY kb-importer-$VULAS_RELEASE.jar kb-importer.jar
 RUN chmod +x kb-importer.jar
 
-COPY kb-importer.sh start.sh /kb-importer/
+COPY start.sh /kb-importer/
+
+EXPOSE 8080
+
 RUN chmod +x /kb-importer/kb-importer.sh /kb-importer/start.sh
```

Comment:
`/kb-importer/kb-importer.sh` is not copied any more, so this can be omitted, correct?

Reply:
Yes